tools/distpartitioning/dataset_utils.py

```diff
@@ -0,0 +1,92 @@
+import os
+import numpy as np
+import constants
+import torch
+
+def get_dataset(input_dir, graph_name, rank, num_node_weights):
+    """
+    Function to read the multiple file formatted dataset. 
+
+    Parameters:
+    -----------
+    input_dir : string
+        root directory where dataset is located.
+    graph_name : string
+        graph name string
+    rank : int
+        rank of the current process
+
+    Return:
+    -------
+    dictionary
+        Data read from nodes.txt file and used to build a dictionary with keys as column names
+        and values as columns in the csv file.
+    dictionary
+        Data read from numpy files for all the node features in this dataset. Dictionary built 
+        using this data has keys as node feature names and values as tensor data representing 
+        node features
+    dictionary
+        Data read from edges.txt file and used to build a dictionary with keys as column names 
+        and values as columns in the csv file. 
+    dicitonary
+        Data read from removed_edges txt file and a dictionary is built with column names as keys
```

Comment:
why do we still have this?

Reply:
This code exists for the following reason: For the "mag" dataset the edges are spread across two files mag_edges.txt and mag_removed_edges.txt. If we remove this code, which handles removed edges separately then the "mag" dataset will no longer work with this pipeline. To make it work, we should either merge the two edges files or renumber the edge_ids in the mag_edges.txt file so that they are contiguous. Please note that because of this change the verify_mag_partitions.py will have to change appropriately as well. 

As suggested, for now, I am removing the code which handles removed edges separately.